### ISSUE TITLE
DMN Runner table columns not visible for large DMN models

### ISSUE THREAD

**Issue by jomarko** [apache/incubator-kie-tools#3769]
This ticket is about DMN Runner table inputs and outputs columns. With the DMN model size, DMN Runner user needs to use  `Splitter` (is highlighted with red in the picture below) to fill in input data or see output data.

![Screenshot 2023-05-17 151922](https://github.com/apache/incubator-kie-tools/assets/8044780/97ec907f-0be0-4078-9f4e-7424999ad082)

However with the growing DMN model size, there is a moment when `Splitter` can not be dragged anymore. It split display in 50:50 ratio. In this situation we could bring an enhancement to suggest user how to see invisible columns.

Currently horizontal scroll can be used, what is browser general feature - 'shift' + 'scroll button', however:
- we do not show horizontal scrollbar anywhere
- we do not show any UI element for 'display more'
- we do not indicate some overlap like we do for a text `...`, I mean some shadow transition in lats column that is visible just partially

### Steps to reproduce
1. Open https://sandbox.kie.org/ and setup kie extended services to be able to run DMN models
2. Import this file (remove .txt extension before importing into sandbox) 
[many-nodes.dmn.txt](https://github.com/kiegroup/kie-issues/files/11498556/many-nodes.dmn.txt)
3. Click run button to open DMN runner and turn on its tabular view
4. Try to fill in `InputData-7` column

**Comment by tiagobento** [comment]
The keyboard shortcuts panel are not related to DMN Runner, but rather only for the DMN Editor.

Also, I don't understand why the splitter should not be draggable. I think we need a better title to discuss what's wrong with the UX.

**Comment by yesamer** [comment]
I agree that the splitter should be draggable, and the current behavior is acceptable from my point of view.
The only friction I see is that it's not obvious to the user that `shift+scroll` scrolls horizontally.
Therefore, I think the aim of this ticket is to improve this part of UX, avoiding the user can feel blocked because he/she is not able to reach a not-visible cell.

**Comment by jomarko** [comment]
@tiagobento @yesamer thank you for your comments. I am totally fine to keep splitter draggable after reading it. However I still kind of feel, it is not straight forward to know, I can use `shift` + `scroll button` to see invisible columns of DMN Runner table. Is that only me? I tried to capture my thought in desription.

If your opinion is that this is widely known browser feature I think we can close this kie-issue apache/incubator-kie-tools#3769. Please let me know.

**Comment by tiagobento** [comment]
Related:
- apache/incubator-kie-tools#3783
